data/tlg0085/tlg001/__cts__.xml

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0085.tlg001" xml:lang="grc" groupUrn="urn:cts:greekLit:tlg0085">
     
-    <ti:title xml:lang="eng">Suppliant Maidens</ti:title>
+    <ti:title xml:lang="grc">Ἱκέτιδες</ti:title>
```

Comment:
@AlisonBabeu 
It's up to you if you want to use the Greek as the canonical title or not. For most of Perseus, I follow the catalog canonical titles. 
If Latin or English is not used, then finding stuff is going to be hard in Scaife until there is a catalog/pinakes interface.